examples/input/gamepad_input.rs

```diff
@@ -19,17 +20,24 @@ struct GamepadLobby {
     gamepad_event_reader: EventReader<GamepadEvent>,
 }
 
+#[derive(Default)]
```

Comment:
I think this example is a good way to test that the implementation works correctly, but I don't think its the best illustration of how people should actually use the api. I'd prefer it if we trimmed it down to be more like the other input examples. The GamepadLobby should probably stay as-is, but can we remove GamepadData (and all "previous value" testing), print pressed, released, and the `Axis<GamepadButton>` axis for a single button (instead of an array), and print a single Axis<GamepadAxis> value? (printing each frame is fine)

Reply:
Now it prints press/release events for South button, value of button RightTrigger2, value for LeftStickX all in the same system `gamepad_system`